doc/src/stdlib/numbers.md

```diff
@@ -102,11 +102,20 @@ The `BigFloat` type implements arbitrary-precision floating-point arithmetic usi
 
 ```@docs
 Base.precision
+Base.MPFR.precision(::Type{BigFloat})
```

Comment:
Does this need to be listed here twice? Doesn't the line before this cause all methods to be documented?